nntrainer/src/conv2d_layer.cpp

```diff
@@ -138,7 +140,8 @@ int Conv2DLayer::setOptimizer(Optimizer &opt) {
   for (iter = list_d.begin(); iter != list_d.end(); ++iter) {
     (*iter).setZero();
   }
-  return this->opt.initialize(list_d, true);
```

Comment:
do we need creation and assignment of list_d now?

Reply:
yes. no need. forgot to remove. thanks.